src/bun.js/bindings/ProcessBindingTTYWrap.cpp

```diff
@@ -394,12 +394,12 @@ class TTYWrapConstructor final : public JSC::InternalFunction {
 
 const ClassInfo TTYWrapConstructor::s_info = { "TTY"_s, &Base::s_info, nullptr, nullptr, CREATE_METHOD_TABLE(TTYWrapConstructor) };
 
-JSValue createBunTTYFunctions(JSC::JSGlobalObject* globalObject)
+JSValue createBunTTYFunctions(Zig::GlobalObject* globalObject)
 {
     auto& vm = globalObject->vm();
     auto* obj = constructEmptyObject(globalObject);
 
-    obj->putDirect(vm, PropertyName(Identifier::fromString(vm, "isTTY"_s)), JSFunction::create(vm, globalObject, 0, "isatty"_s, Zig::jsFunctionTty_isatty, ImplementationVisibility::Public), 0);
+    obj->putDirect(vm, PropertyName(Identifier::fromString(vm, "isatty"_s)), JSFunction::create(vm, globalObject, 0, "isatty"_s, Zig::jsFunctionTty_isatty, ImplementationVisibility::Public), 0);
```

Comment:
☝️ This fix a different bug?

Reply:
no it just matches the name we actually assign this function into at the binding

<img width="1409" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/e24002b1-c3d5-45b6-8eca-03e301fdb1aa">